AppCenterCrashes/AppCenterCrashesTests/MSACErrorLogFormatterTests.mm

```diff
@@ -766,97 +764,41 @@ - (void)checkIfArchNameConvertedCorrectly:(NSNumber *)codeType archName:(NSNumbe
-  [self checkIfCodeTypeConvertedCorrectly:@(CPU_TYPE_ARM) isKnownEncodingType:YES expectedType:@"ARM"];
-  [self checkIfCodeTypeConvertedCorrectly:@(CPU_TYPE_ARM64) isKnownEncodingType:YES expectedType:@"ARM-64"];
-  [self checkIfCodeTypeConvertedCorrectly:@(CPU_TYPE_X86) isKnownEncodingType:YES expectedType:@"X86"];
-  [self checkIfCodeTypeConvertedCorrectly:@(CPU_TYPE_X86_64) isKnownEncodingType:YES expectedType:@"X86-64"];
-  [self checkIfCodeTypeConvertedCorrectly:@(CPU_TYPE_POWERPC) isKnownEncodingType:YES expectedType:@"PPC"];
+  [self checkIfCodeTypeConvertedCorrectly:@(CPU_TYPE_ARM) expectedType:@"ARM"];
+  [self checkIfCodeTypeConvertedCorrectly:@(CPU_TYPE_ARM64) expectedType:@"ARM-64"];
+  [self checkIfCodeTypeConvertedCorrectly:@(CPU_TYPE_X86) expectedType:@"X86"];
+  [self checkIfCodeTypeConvertedCorrectly:@(CPU_TYPE_X86_64) expectedType:@"X86-64"];
+  [self checkIfCodeTypeConvertedCorrectly:@(CPU_TYPE_POWERPC) expectedType:@"PPC"];
 }
 
 - (void)testConvertArchNameToString {
   [self checkIfArchNameConvertedCorrectly:@(CPU_TYPE_ARM)
                              archName:@(CPU_SUBTYPE_ARM_V6)
-                  isKnownEncodingType:YES
                          expectedName:@"armv6"];
   [self checkIfArchNameConvertedCorrectly:@(CPU_TYPE_ARM)
                              archName:@(CPU_SUBTYPE_ARM_V7)
-                  isKnownEncodingType:YES
                          expectedName:@"armv7"];
   [self checkIfArchNameConvertedCorrectly:@(CPU_TYPE_ARM)
                              archName:@(CPU_SUBTYPE_ARM_V7S)
-                  isKnownEncodingType:YES
                          expectedName:@"armv7s"];
   [self checkIfArchNameConvertedCorrectly:@(CPU_TYPE_ARM64)
                              archName:@(CPU_SUBTYPE_ARM64_ALL)
-                  isKnownEncodingType:YES
                          expectedName:@"arm64"];
   [self checkIfArchNameConvertedCorrectly:@(CPU_TYPE_ARM64)
                              archName:@(CPU_SUBTYPE_ARM64_V8)
-                  isKnownEncodingType:YES
                          expectedName:@"armv8"];
   [self checkIfArchNameConvertedCorrectly:@(CPU_TYPE_ARM64)
                              archName:@(CPU_SUBTYPE_ARM64E)
-                  isKnownEncodingType:YES
                          expectedName:@"arm64e"];
   [self checkIfArchNameConvertedCorrectly:@(CPU_TYPE_X86)
                              archName:@(CPU_TYPE_X86)
-                  isKnownEncodingType:YES
                          expectedName:@"i386"];
   [self checkIfArchNameConvertedCorrectly:@(CPU_TYPE_X86_64)
                              archName:@(CPU_TYPE_X86_64)
-                  isKnownEncodingType:YES
                          expectedName:@"x86_64"];
   [self checkIfArchNameConvertedCorrectly:@(CPU_TYPE_POWERPC)
                              archName:@(CPU_TYPE_POWERPC)
-                  isKnownEncodingType:YES
                          expectedName:@"powerpc"];
 }
 
-- (void)testConvertCodeTypeUnknownEncodingTypeToString {
```

Comment:
What about the case when archName is not found? This line https://github.com/microsoft/appcenter-sdk-apple/blob/5105fc4ff7dba9238546a170b090ef4b0ac2415e/AppCenterCrashes/AppCenterCrashes/Internals/Util/MSACErrorLogFormatter.m#L473.